src/server/dfly_bench.cc

```diff
@@ -741,17 +741,21 @@ void Driver::ParseRESP() {
   uint32_t consumed = 0;
   RedisParser::Result result = RedisParser::OK;
   RespVec parse_args;
+  constexpr string_view kMovedErrorKey = "-MOVED"sv;
 
   do {
     result = parser_.Parse(io_buf_.InputBuffer(), &consumed, &parse_args);
     if (result == RedisParser::OK && !parse_args.empty()) {
       if (parse_args[0].type == RespExpr::ERROR) {
         string_view error = io::View(io_buf_.InputBuffer());
         VLOG(2) << "Error " << error;
-        if (absl::StartsWith(error, "-MOVED ")) {
-          error = error.substr(7);
+        if (absl::StartsWith(error, kMovedErrorKey)) {
```

Comment:
if you are at it, please change line 750 to: `string_view error = parse_args[0].View();` because otherwise this code breaks once dfly_bench is run with --pipeline=10